tools/run_tests/python_utils/upload_rbe_results.py

```diff
@@ -287,7 +287,8 @@ def _get_resultstore_data(api_key, invocation_id):
 
     if not args.skip_upload:
         # BigQuery sometimes fails with large uploads, so batch 1,000 rows at a time.
-        for i in range((len(bq_rows) / 1000) + 1):
-            _upload_results_to_bq(bq_rows[i * 1000:(i + 1) * 1000])
+        MAX_ROWS = 1000
```

Comment:
can you explain what what was the erroneous behavior before? I don't see the issue with the old code right away so it's useful to keep record of what the problem was.

Was the problem that the code didn't work if len(bq_rows) was exacly divisible by 1000?

Reply:
Yes, when it's divisible by 1000, it will have _upload_results_to_bq called with an empty list.